src/main/java/nextstep/path/SecondDistanceFare.java

```diff
@@ -0,0 +1,24 @@
+package nextstep.path;
+
+public class SecondDistanceFare extends DistanceFare {
+
+    public SecondDistanceFare(int distance) {
+        super(distance);
+    }
+
+    @Override
+    public int calculateFare() {
+        int remain = getRemain();
+        if (remain == 0 || distance <= 10) {
+            return 0;
+        }
+        return (int) ((Math.ceil((remain - 1) / 5) + 1) * 100);
+    }
+
+    private int getRemain() {
+        if (distance > 50) {
+            return 40;
+        }
+        return distance - 10;
```

Comment:
```suggestion
        return Math.max(40, distance - 10);
```
이런 방식의 표현도 도움이 될 것 같습니다 :)